man/docker-cp.1.md

```diff
@@ -147,6 +148,17 @@ to a directory `/etc/my-app.d` in a container:
 The above command will copy the contents of the local `/config` directory into
 the directory `/etc/my-app.d` in the container.
 
+Finally, if you want to copy a symbolic link into a container, you typically
+want to  copy the linked target and not the link itself. To copy the target, use
+the `-L` option, for example:
+
+		$ ln -s /tmp/somefile /tmp/somefile.ln
+		$ docker cp -L /tmp/somefile.ln myappcontainer:/tmp/
```

Comment:
Looks like these lines are indented one tab too much. Also, could you change to 4 spaces in stead of tabs?